docs/production-setup.md

```diff
@@ -81,7 +81,7 @@ fvm flutter build ipa --flavor Bayern --release --dart-define=environment=produc
 #### iOS:
 - `open ./build/ios/archive/Runner.xcarchive`
 - use manual signing and choose match appstore certificate
-- navigate the dialog and finally distribute the app. This will create a release on Testflight. 
+- navigate the dialog and finally distribute the app. This will create a release on Testflight. It may take a while until the new build will be listed.
```

Comment:
Sorry, but this bothers me 😅 
```suggestion
- Navigate the dialog and finally distribute the app. This will create a release on Testflight. It may take a while until the new build will be listed.
```